exp/api/v1alpha3/awsmanagedmachinepool_types.go

```diff
@@ -129,8 +129,10 @@ type ManagedRemoteAccess struct {
 	SSHKeyName *string `json:"sshKeyName,omitempty"`
 
 	// SourceSecurityGroups specifies which security groups are allowed access
-	// An empty array opens port 22 to the public internet
 	SourceSecurityGroups []string `json:"sourceSecurityGroups,omitempty"`
+
+	// Public specifies whether to open port 22 to the public internet
+	Public bool `json:"public,omitempty"`
```

Comment:
Small nit, in the PR description its `publicAccess` but here its `public`. Guessing the preferred naming is public?

Reply:
Yes, I switched to `public` to get rid of the `remoteAccess.publicAccess` duplication.